scripts/pre-release-vars.sh

```diff
@@ -0,0 +1,2 @@
+export BRANCH=main
+export CURRENT_SNAPSHOT_VERSION=1.0.0.Final-SNAPSHOT
```

Comment:
@marcosgopen I think this should be 0.0.9.Final-SNAPSHOT?

Reply:
Thanks, I missed that.